pymc/initial_point.py

```diff
@@ -124,7 +129,7 @@ def make_initial_point_fn(
     jitter_rvs: set[TensorVariable] | None = None,
     default_strategy: str = "support_point",
     return_transformed: bool = True,
-) -> Callable:
+) -> Callable[[int], PointType]:
```

Comment:
```suggestion
) -> Callable[[int], PointType]:
```
This is the right type hint for the input: https://github.com/pymc-devs/pymc/blob/59aa6e2ceda3e878873ffa9ef25d3b4772ff0adb/pymc/pytensorf.py#L798